helm/libero-editor/Chart.yaml

```diff
@@ -2,4 +2,4 @@ apiVersion: v1
 appVersion: "latest"
 description: eLife specific deployment of Libero Editor
 name: libero-editor
-version: 0.0.8
+version: 0.0.9
```

Comment:
Worried this might break something with current staging 🤷‍♂️

Reply:
Editor isn't a critical system, so no need to panic if things don't work first time :-)